spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -3,25 +3,25 @@
 describe DispatchRider::Subscriber do
 
   before do
-    DispatchRider::Handlers::Base.stub(:subclasses){ Set.new }
+    allow(DispatchRider::Handlers::Base).to receive(:subclasses) { Set.new }
 
     stub_const("FooBar", Class.new(DispatchRider::Handlers::Base) {
-      def process(options)
-        throw :process_was_called
-      end
-    })
+                         def process(options)
+                           throw :process_was_called
+                         end
+                       })
```

Comment:
`end` at 12, 23 is not aligned with `Class.new(DispatchRider::Handlers::Base) {` at 8, 25 or `stub_const("FooBar", Class.new(DispatchRider::Handlers::Base) {` at 8, 4